draft-ietf-quic-transport.md

```diff
@@ -1979,7 +1979,10 @@ to verify the peer's ownership of the unvalidated address.
 An endpoint MAY send data to an unvalidated peer address, but it MUST protect
 against potential attacks as described in {{address-spoofing}} and
 {{on-path-spoofing}}.  An endpoint MAY skip validation of a peer address if that
-address has been seen recently.
+address has been seen recently.  In particular, if an endpoint returns to a
+previously-validated path after detecting some form of spurious migration,
+skipping address validation and restoring loss detection and congestion state
+can reduce the performance impact of the attack.
```

Comment:
The way I think about this is that the old address validation and congestion state are maintained until the new address is validated.  ie: In my thinking, the responder never left the original path.

Reply:
Yeah, that's harder to write down, I think.